incubator/elasticsearch/values-production.yaml

```diff
@@ -0,0 +1,211 @@
+## Bitnami Elasticsearch image version
+## ref: https://hub.docker.com/r/bitnami/elasticsearch/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/elasticsearch
+  tag: 6.2.3-r3
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
```

Comment:
@sameersbn, as we are using a rolling tag now, I think we should set the pullPolicy to `Always` don't you think?

Reply:
Yes! You are right.

Reply:
👍